package.json

```diff
@@ -1,26 +1,25 @@
 {
   "name": "@mparticle/web-rokt-wsdk-kit",
```

Comment:
i'd proposing dropping the `wsdk`, as this aligns with our other kits

Reply:
![image](https://github.com/user-attachments/assets/7700df16-0ad5-4666-ab4d-4b731b393628)